static/js/composebox_typeahead.js

```diff
@@ -449,6 +450,10 @@ export function get_person_suggestions(query, opts) {
         if (opts.want_broadcast) {
             persons = persons.concat(broadcast_mentions());
         }
+
+        // Exclude muted users from typeaheads.
+        persons = muting.filter_muted_users(persons);
```

Comment:
I think we probably want this to be before `broadcast_mentions`, since those aren't people.